Makefile

```diff
@@ -60,6 +60,6 @@ setup:
 test:
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
 	$(ISTANBUL) check-coverage --branches 100
-ifneq ($(shell node --version | sed 's/[.][^.]*$$//'),v0.10)
+ifeq ($(shell node --version | cut -d . -f 1),v6)
```

Comment:
Actually, I decided to restrict myself to ASCII regardless of environment. I had too much trouble getting CircleCI to do the right thing. Also, I'm now redirecting the output to stderr.